ci/docker/runtime_functions.sh

```diff
@@ -381,13 +381,12 @@ unittest_ubuntu_python3_cpu() {
 
 unittest_ubuntu_python3_cpu_mkldnn() {
     set -ex
-    export PYTHONPATH=./python/ 
+    export PYTHONPATH=./python/
     # MXNET_MKLDNN_DEBUG is buggy and produces false positives
     # https://github.com/apache/incubator-mxnet/issues/10026
     #export MXNET_MKLDNN_DEBUG=1  # Ignored if not present
     export MXNET_STORAGE_FALLBACK_LOG_VERBOSE=0
     nosetests-3.4 --verbose tests/python/unittest
-    nosetests-3.4 --verbose tests/python/quantization
```

Comment:
One note for removing "nosetests-3.4 --verbose tests/python/quantization" for unittest_ubuntu_python3_cpu_mkldnn: this is due to we used tests/python/mkl/test_quantization_mkldnn.py (will be tested when running "nosetests-3.4 --verbose tests/python/mkl") to test mkldnn quantization so could remove tests/python/quantization path.